### PR TITLE
Update to 1.8.4 + new skin + upgrade plugins

### PR DESCRIPTION
**Pull request by maniackcrudelis**
## Problem
- *New package version upstream*

## Solution
- *Upgrade NodeJS to v.12*
- *Set colibris as skin default*
- *Add skin variants in settings*
- *upgrade packages to upstream*
- *Fix word caps*

## PR Status
- [x] Code finished.
- [x] Tested with Package_check.
- [x] Fix or enhancement tested.
- [x] Upgrade from last version tested.
- [x] Can be reviewed and tested.

## Validation
---
*Minor decision*
- [ ] **Code review** : 
- [ ] **Approval (LGTM)** :  
*Code review and approval have to be from a member of @YunoHost-Apps/apps-group*
- **CI succeeded** : 
[Build Status](https://ci-apps-hq.yunohost.org/jenkins/job/etherpad_mypads_ynh%20PR105/)  
When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

### CODE REVIEW

**Comment by zamentur** [comment]
I tested it . It seems ok (but we exprement currently some 502 and it's not completely stable. It could be due to the load of our instances.

The current 1.8.0 in stable branch have some serious issues too.

Here the issues with this testing: https://paste.yunohost.org/naqiyubugi.coffeescript

There are also this error:
```
Error: BACKEND.ERROR.CONFIGURATION.KEY_NOT_FOUND
    at Object.callback (/var/www/etherpad_mypads/node_modules/ep_mypads/model/common.js:176:25)
    at /var/www/etherpad_mypads/src/node_modules/ueberdb2/CloneAndAtomicLayer.js:99:17
    at exports.database.get (/var/www/etherpad_mypads/src/node_modules/ueberdb2/CacheAndBufferLayer.js:149:5)
    at exports.channels.doOperation [as operatorFunction] (/var/www/etherpad_mypads/src/node_modules/ueberdb2/CloneAndAtomicLayer.js:93:18)
    at exports.channels.emit (/var/www/etherpad_mypads/src/node_modules/channels/channels.js:49:11)
    at exports.database.get (/var/www/etherpad_mypads/src/node_modules/ueberdb2/CloneAndAtomicLayer.js:61:17)
    at Object.module.exports.common.getDel (/var/www/etherpad_mypads/node_modules/ep_mypads/model/common.js:173:16)
    at Object.wrapper [as get] (/var/www/etherpad_mypads/node_modules/lodash/index.js:3592:19)
    at Object.module.exports.perm.fn.getPadAndGroup (/var/www/etherpad_mypads/node_modules/ep_mypads/perm.js:113:9)
    at module.exports.perm.check (/var/www/etherpad_mypads/node_modules/ep_mypads/perm.js:272:15)
```

File: scripts/install [hunk 1]
```diff
@@ -242,37 +242,37 @@ ynh_script_progression --message="Installing Etherpad plugins..." --weight=90
...
+npm install ep_spellcheck@${ep_spellcheck_version} >> $install_log 2>&1
 # Framapad - Add support for Subscript and Superscript
-npm install ep_subscript_and_superscript >> $install_log 2>&1
+npm install ep_subscript_and_superscript@${ep_subscript_and_superscript_version} >> $install_log 2>&1
 # Framapad - View a table of contents for your pad
-npm install ep_table_of_contents >> $install_log 2>&1
+npm install ep_table_of_contents@${ep_table_of_contents_version} >> $install_log 2>&1
 # Framapad - User Pad Contents font size can be set in settings, this does not effect other peoples views
-npm install ep_user_font_size >> $install_log 2>&1
+npm install ep_user_font_size@${ep_user_font_size_version} >> $install_log 2>&1
```
Comment (alexAubin) [inline scripts/install:275]: I guess that works, but it really sounds like we're reinventing packages.json here...

File: scripts/_variables [hunk 2]
```diff
@@ -11,4 +11,20 @@ nodejs_version=10
 
 # Mypads version
 # This variable is mostly used to force an upgrade of the package in case of new versions of mypads.
-mypads_version=1.7.10
+mypads_version=1.7.17
+
+# Plugin versions
+ep_align_version=0.2.0
+ep_author_hover_version=0.2.1
+ep_automatic_logut_version=1.0.8
```
Comment (alexAubin) [inline scripts/_variables:19]:
```suggestion
ep_automatic_logout_version=1.0.8
```

File: scripts/config [hunk 3]
```diff
@@ -236,7 +236,7 @@ apply_config() {
             npm uninstall ep_automatic_logut
         else
             ynh_replace_string --match_string="^\/\/\(.*\"automatic_logut.*$\)" --replace_string="\1" --target_file="$config_file"
-            npm install ep_automatic_logut
+            npm install ep_automatic_logut@${ep_automatic_logut_version}
```
Comment (alexAubin) [inline scripts/config:239]:
```suggestion
            npm install ep_automatic_logut@${ep_automatic_logout_version}
```

**Review by alexAubin — DISMISSED** [review]
Proposing to merge soonish if no complains...

File: scripts/install (hunk 1 above)
Comment (kay0u) [reply scripts/install:275]: IIRC Maniack had a lot of problems with the packages.json on this app, and the developer was not very cooperative with him.

**Comment by kay0u** [comment]
I'm ok with that

**Pull request merged**